ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml

```diff
@@ -21,6 +21,8 @@ spec:
       description: Repository name assigned to certification project from Red Hat Connect
     - name: dist_method
```

Comment:
This is no longer used and can be removed

Reply:
Removed